dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java

```diff
@@ -174,8 +175,12 @@
         viewAsFields.put("mode", new TypeFetcher(GraphQLString,
                 PropertyDataFetcher.fetching((Function<ViewAsPageStatus, String>)
                         (viewAs)->viewAs.getPageMode().name())));
-        viewAsFields.put("persona", new TypeFetcher(GraphQLTypeReference.typeRef("PersonaBaseType"),
-                new PropertyDataFetcher<ViewAsPageStatus>("persona")));
+        if(LicenseManager.getInstance().isEnterprise()) {
+            viewAsFields
+                    .put("persona", new TypeFetcher(GraphQLTypeReference.typeRef("PersonaBaseType"),
```

Comment:
Codacy found an issue: [The String literal "persona" appears 5 times in this file; the first occurrence is on line 163](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190817)